tests/cephadm/test_legacy_services.py

```diff
@@ -0,0 +1,55 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify if the cephadm shell tries to infer config from
+    a dummy mon created.
+    Verifies bz:2080242
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        **kw :     Key/value pairs of configuration information to be used in the test.
+
+    Returns:
+        int - 0 when the execution is successful else 1 (for failure).
+    """
+
+    config = kw.get("config")
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+
+    all_nodes = cephadm.cluster.get_nodes()
+    mon_nodes = cephadm.cluster.get_nodes(role="mon")
+    client_node = cephadm.cluster.get_nodes(role="client")
+    # Exclude the client node
+    mon_nodes.extend(client_node)
+
+    # Create dummy mon on all non-mon nodes
+    for node in set(all_nodes) - set(mon_nodes):
+        # Create a dummy mon on a non monitor node
+        hostname = node.hostname
+        dummy_mon = f"sudo touch /var/lib//ceph/mon/ceph-{hostname}"
```

Comment:
`/var/lib//ceph/mon` ??

Reply:
That was a typo, will change. The additional '/' hasn't caused any issues in touch ops.
eg. 
prprakas@localhost $ mkdir /tmp//test
prprakas@localhost $ ls /tmp | grep test
test

Will remove the additional '/'. Thanks for pointing it. :)

Reply:
@manasagowri used the same way while verifying bz are you sure it's typo?

Reply:
@sunilangadi2 IMO I feel its a typo as it has not impact in the file path specified.
https://stackoverflow.com/questions/41856552/what-does-double-slash-mean-in-the-file-path